consul_connect/README.md

```diff
@@ -0,0 +1,80 @@
+# Agent Check: Consul Connect
+
+## Overview
+
+Monitor your [Consul Connect][1] Envoy sidecar proxies with the [Datadog Envoy Integration][2]. Although Consul Connect has support for external proxies, this setup refers to [Consul Connect configured with Envoy][3]. 
+
+## Setup
+
+### Installation
+
+Install the Datadog Agent on your services running Consul Connect and follow the [Configuration](#configuration) instructions for your appropriate environment.
+
+### Configuration
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Host" xxx -->
+
+#### Host
+
+To configure this check for an Agent running on a host:
+
+##### Metric Collection
+Use config option [`-admin-bind`][4] in Consul Connect to ensure the Envoy Admin API is exposed, then follow the [Envoy host instructions][5] to configure metric collection.
```

Comment:
Is the default sufficient or are you suppose to set `-admin-bind` to a different port?